### ISSUE TITLE
NullPointerException(9.10.1)

### ISSUE THREAD

**Issue by jhwsx** [AppLovin/Android-SDK-Demo#52]
I have updated sdk version to 9.10.1, however, a lot of NPE pointing to this sdk have been happening. The log is as follows:
```text
java.lang.RuntimeException: Unable to start activity ComponentInfo{com.sail.advanced.booster/com.applovin.adview.AppLovinInterstitialActivity}: java.lang.NullPointerException: Attempt to invoke virtual method &#39;int com.applovin.impl.sdk.ad.f.V()&#39; on a null object reference
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2790)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2855)
	at android.app.ActivityThread.access$900(ActivityThread.java:181)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1474)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:145)
	at android.app.ActivityThread.main(ActivityThread.java:6117)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
Caused by: java.lang.NullPointerException: Attempt to invoke virtual method &#39;int com.applovin.impl.sdk.ad.f.V()&#39; on a null object reference
	at com.applovin.impl.adview.m.dismiss(Unknown Source)
	at com.applovin.impl.adview.r.dismiss(Unknown Source)
	at com.applovin.impl.adview.m.exitWithError(Unknown Source)
	at com.applovin.impl.adview.m.onCreate(Unknown Source)
	at com.applovin.impl.adview.r.onCreate(Unknown Source)
	at android.app.Activity.performCreate(Activity.java:6374)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2743)
	... 10 more
java.lang.NullPointerException: Attempt to invoke virtual method &#39;int com.applovin.impl.sdk.ad.f.V()&#39; on a null object reference
	at com.applovin.impl.adview.m.dismiss(Unknown Source)
	at com.applovin.impl.adview.r.dismiss(Unknown Source)
	at com.applovin.impl.adview.m.exitWithError(Unknown Source)
	at com.applovin.impl.adview.m.onCreate(Unknown Source)
	at com.applovin.impl.adview.r.onCreate(Unknown Source)
	at android.app.Activity.performCreate(Activity.java:6374)
	at android.app.Instrumentation.callActivityOnCreate(Instrumentation.java:1119)
	at android.app.ActivityThread.performLaunchActivity(ActivityThread.java:2743)
	at android.app.ActivityThread.handleLaunchActivity(ActivityThread.java:2855)
	at android.app.ActivityThread.access$900(ActivityThread.java:181)
	at android.app.ActivityThread$H.handleMessage(ActivityThread.java:1474)
	at android.os.Handler.dispatchMessage(Handler.java:102)
	at android.os.Looper.loop(Looper.java:145)
	at android.app.ActivityThread.main(ActivityThread.java:6117)
	at java.lang.reflect.Method.invoke(Native Method)
	at java.lang.reflect.Method.invoke(Method.java:372)
	at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1399)
	at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1194)
```
I have read the log carefully, none of the lines points to our own codes.

**Comment by thomasmso** [comment]
Hi, this has been fixed in our latest Android SDK. Please feel free to follow [this repo](https://github.com/AppLovin/AppLovin-MAX-SDK-Android) to get updates on our SDK releases.

**Comment by jhwsx** [comment]
BTW, we integrate AppLovin sdk by Mopub mediation. My question is: 
Mopub writes we should use:
```gradle
// AppLovin
implementation 'com.applovin:applovin-sdk:9.10.2'
implementation 'com.mopub.mediation:applovin:9.10.2.0'
```
We are going to use 9.10.2 version to avoid mopub incompatible or strange issue.
But 9.10.2 is not the latest AppLovin sdk, has 9.10.2 sdk ever fixed this NPE issue?

**Comment by thomasmso** [comment]
9.10.2 fixed the NPE but it's safe to just go on 9.10.3.
